.ddev/config.yaml

```diff
@@ -0,0 +1,39 @@
+# DDev configuration
+# @see https://ddev.readthedocs.io/en/latest/users/extend/config_yaml/
+
+# Update to match your project name. Using "drupal-skeleton" would make the site
+# accessible at 'drupal-skeleton.ddev.site'.
+name: drupal-skeleton
+
+# Use 'docroot' for Acquia, or 'web' for Pantheon or Platform.sh.
+docroot: web
+
+# Add any multisite subdomains to this array. Adding "example" will make this environment
+# available at "example.ddev.site".
+additional_hostnames: []
+
+# Add extra fully qualified domains here. This is useful for transitioning from a vagrant
+# environment to a ddev environment.
+additional_fqdns: []
+
+# The PHP version should match the project's hosting environment.
+php_version: "7.3"
+
+# It may make sense to turn this off once the project is set up.
+disable_settings_management: false
+
+# Turn XDebug on and off for your own environment using `ddev xdebug on` and
+# `ddev xdebug off`
+xdebug_enabled: false
+
+# NFS speeds up the ddev environment and makes file changes sync more consistently.
+# @see https://ddev.readthedocs.io/en/latest/users/performance/#using-nfs-to-mount-the-project-into-the-web-container
+nfs_mount_enabled: false
+
+type: drupal9
+webserver_type: nginx-fpm
+router_http_port: "80"
+router_https_port: "443"
+mariadb_version: "10.3"
+provider: default
+use_dns_when_possible: true
```

Comment:
@becw should we add nvm?
```
# Install nvm and use Node 10 (or whatever version you choose).
hooks:
 post-start:
   - exec: curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.11/install.sh | bash
   - exec: rm -f ../.nvmrc && export NVM_DIR="$HOME/.nvm" && source "$NVM_DIR/nvm.sh" && nvm install 10 && nvm use 10
```
(From https://stackoverflow.com/questions/58415512/how-can-i-add-and-use-nvm-in-a-ddev-web-container)

When I first tried this config on an existing site, I found they required Node `>=6 <11`, but by default, this had version 14.

Reply:
Yes, this would be great! Is there a good way to specify the node version in the styleguide, and then tell nvm to use that version?

Reply:
That's a great idea! I'll look into this.